src/acceptance/app/dynamic_policy_test.go

```diff
@@ -43,29 +50,39 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 		guid := cf.Cf("app", appName, "--guid").Wait(cfg.DefaultTimeoutDuration())
 		Expect(guid).To(Exit(0))
 		appGUID = strings.TrimSpace(string(guid.Out.Contents()))
-
 		Expect(cf.Cf("start", appName).Wait(cfg.CfPushTimeoutDuration())).To(Exit(0))
 		WaitForNInstancesRunning(appGUID, initialInstanceCount, cfg.DefaultTimeoutDuration())
+
+		if cfg.IsServiceOfferingEnabled() {
+			bindService := cf.Cf("bind-service", appName, instanceName, "-c", policy).Wait(cfg.DefaultTimeoutDuration())
+			Expect(bindService).To(Exit(0), "failed binding service to app with a policy ")
+		} else {
+			oauthToken = OauthToken(cfg)
```

Comment:
how about define a function for attaching policy so all the test suites can reference it?